quint/src/IRprinting.ts

```diff
@@ -23,7 +23,7 @@ import { QuintType, Row } from './quintTypes'
  * @returns a string with the pretty printed definition
  */
 export function moduleToString(quintModule: QuintModule): string {
-  const defs = quintModule.defs.map(d => definitionToString(d)).join('\n  ')
+  const defs = quintModule.defs.map((d) => definitionToString(d)).join('\n  ')
```

Comment:
Can we try to disable this rule? I dislike it.

Reply:
Disables in 65f662240706d57f5a967fbcfb08c47122addf91

Reply:
Thank you :tada: